<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportPage1</class>
 <widget class="QWidget" name="ExportPage1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout">
   <item row="9" column="0">
    <widget class="QLabel" name="lblPacketNumber">
     <property name="text">
      <string>Номер пакета</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QComboBox" name="cmbRegistryType"/>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="chkVerboseLog">
     <property name="text">
      <string>Подробный отчет</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QVBoxLayout" name="vlOptions"/>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="chkIgnoreErrors">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Игнорировать ошибки</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="3">
    <widget class="CProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="3">
    <widget class="QLabel" name="lblElapsed">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QSpinBox" name="edtPacketNumber"/>
   </item>
   <item row="13" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblRevision">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>прервать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExport">
       <property name="text">
        <string>экспорт</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0" colspan="3">
    <widget class="QTextBrowser" name="logBrowser"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lblRegistryType">
     <property name="text">
      <string>Тип реестра</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CProgressBar</class>
   <extends>QProgressBar</extends>
   <header>library/ProgressBar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
